lambda/main.go

```diff
@@ -98,6 +107,11 @@ func Handler(ctx context.Context, evt json.RawMessage) (string, error) {
 		return "", err
 	}
 
+	original, ok := b.(*backend.NewRelicBackend)
```

Comment:
This is totally fine for now, in future we could add a `Closer` interface or similar to avoid the type specific cast.